presto-hive-hadoop2/bin/start_hive.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -euo pipefail
+
+. ${BASH_SOURCE%/*}/common.sh
+
+cleanup_docker_containers
+start_docker_containers
+
+HADOOP_MASTER_IP=$(hadoop_master_ip)
+CONTAINER=$(echo $HADOOP_MASTER_CONTAINER | cut -b1-12)
```

Comment:
```suggestion
CONTAINER=$(echo "${HADOOP_MASTER_CONTAINER}" | cut -b1-12)
```

Reply:
what is this `cut -b1-12` for?

Reply:
This shortens the huge identifier to the first 12 characters, which matches what tools like `docker ps` show. There might be a better way to do this.

Reply:
That variable will always be an alphanumeric string (Docker container identifier). Does it still need quoting?

Reply:
> Does it still need quoting?

Technically no. I'd treat it as a matter of code style.

Reply:
> This shortens the huge identifier to the first 12 characters, which matches what tools like `docker ps` show.

Would you mind leaving a word of explanation in the code?

Reply:
Done